scriptmodules/libretrocores/lr-mesen.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="lr-mesen"
+rp_module_desc="High-accuracy NES and Famicom emulator"
+rp_module_help="ROM Extensions: .nes .fds .unf .unif .zip\n\nCopy your NES roms to $romdir/nes\nFamicom roms to $romdir/fds\nCopy the recommended BIOS file disksys.rom to $biosdir"
+rp_module_licence="GPL3 https://raw.githubusercontent.com/sourmesen/mesen/master/LICENSE"
+rp_module_section="exp"
+rp_module_flags="!armv6"
+
+function sources_lr-mesen() {
+    gitPullOrClone "$md_build" https://github.com/SourMesen/Mesen.git
+}
+
+function build_lr-mesen() {
+    make -C Libretro
```

Comment:
Should have a make clean also (for now - I may change this for all modules in the future).

Reply:
done